android/build.gradle

```diff
@@ -60,6 +60,6 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.getphyllo:phyllo-connect:0.3.0" 
-    //implementation "com.getphyllo:phyllo-connect:0.2.5-SNAPSHOT"   
+    //implementation "com.getphyllo:phyllo-connect:0.3.0"
+    implementation "com.getphyllo:phyllo-connect:0.3.2.5-SNAPSHOT"   
```

Comment:
@erpankajpatel , once the android goes, live we will update it to original version rather than SNAPSHOT?

Reply:
@chiragphyllo once we will merge from release_v0.3.2 to main it will be change after release Android SDK .

Reply:
Is Android SDK not released yet with analytics changes? If not, we should hold this release also since it will be hard to get users to update their SDKs again and again because of our internal changes.